base_name_search_improved/__openerp__.py

```diff
@@ -4,7 +4,7 @@
 {
     'name': 'Improved Name Search',
     'summary': 'Friendlier search when typing in relation fields',
-    'version': '8.0.1.0.1',
```

Comment:
We reviewed the module,  now in case of limit false behaves as one would expect.
Why change the version name? There are no migration scripts. It

Reply:
The module version number doesn't change because of migration scripts, but to note that this is something different, so you can say, version 8.0.1.0.1 fails because of that error, but check if you have version x that doesn't have that problem. See https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers for reference.